src/foam/nanos/notification/NotificationView.js

```diff
@@ -8,24 +8,43 @@ foam.CLASS({
   package: 'foam.nanos.notification',
   name: 'NotificationView',
   extends: 'foam.u2.View',
-  axioms: [ foam.pattern.Faceted.create() ],
 
-  properties: [ 'of' ],
+  axioms: [
+    foam.pattern.Faceted.create()
+  ],
+
+  properties: [
+    'of',
+    {
+      class: 'Boolean',
+      name: 'fullyVisible',
+      documentation: `If a notification's body content is too long, it will be
+          truncated and an ellipsis will be added to the cutoff point. Clicking
+          on a notification toggles between truncated form and being fully
+          visible. This property tracks whether the notification is being
+          truncated or displayed in full.`,
+      value: false
+    }
+  ],
+
   methods: [
     function initE() {
       this.SUPER();
       this
         .addClass(this.myClass())
-        .start('div').on('click', this.onClick).addClass('msg').add(this.data.body).end();
+        .start('div')
```

Comment:
`.start()` defaults to div. Passing in 'div' as an argument is somewhat redundant... Just a side note.

Reply:
Thanks, fixed.